Assets/_Laz/Scripts/Managers/WorldManager.cs

```diff
@@ -48,11 +48,11 @@ public void CleanUp()
                 patrolItem.CleanUp();
             }
         }
-        
+
         public void Reset()
         {
             _wrappableManager.Reset();
-            _lazCoordinator.Reset();
+            _lazCoordinator.Reset(_checkPointManager.GetActiveCheckpointPosition());
```

Comment:
Instead of sending in the active checkpoint position each and every time, you can pass in the whole `_checkPointManager` when binding stuff together

Reply:
so itl'l be 

_lazCoordinator.initialzie(...,..., _checkpointManager)